source/GameData.cpp

```diff
@@ -652,6 +652,21 @@ const Set<Planet> &GameData::Planets()
 
 
 
+// Playlists for playing in-game music.
+const Set<Playlist> &GameData::Playlists()
+{
+	return objects.playlists;
+}
+
+
+
+bool GameData::HasPlaylists()
+{
+	return objects.playlists.empty();
```

Comment:
```suggestion
	return !objects.playlists.empty();
```
am I confused or is this a fix?

Reply:
May be a fox but I need to look through the code to verify what else needs to change.  I play tested it and from a data file playtime perspective the code works so likely changing this means I'll need to invert it in other places

Reply:
I still need to look through and verify this.

It is needed for unreleased features.  Overall, this feature is a starting point and not the end of the maturity of the endless sky music engine.  Some of the implementation was made here anticipating its need based on implementation of other sub-systems within ES code base and how I think content creators would use music as part of writing stories.

Reply:
function has been removed